azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -928,7 +928,7 @@ public void cancelExecutingJobs(ExecutableFlow exFlow, String userId,
     }
   }
 
-  private void applyDisabledJobs(List<Object> disabledJobs,
+  public static void applyDisabledJobs(List<Object> disabledJobs,
```

Comment:
I think it's fine. IMO the class contains a few util method, and they are supposed to be declared as static as well.